Source/Bogus/Randomizer.cs

```diff
@@ -41,7 +41,10 @@ public Randomizer(int localSeed)
          this.localSeed = new Random(localSeed);
       }
 
-      private readonly Random localSeed;
+      /// <summary>
+      /// The pseudo-random number generator that is used for all random number generation in this instance.
+      /// </summary>
+      protected Random localSeed;
```

Comment:
It surprised me that I couldn't keep `localSeed` as readonly.  But from what I'm reading in the C# specification, the readonly applies only to the ctor of the class where it is declared.  (i.e. it can't be set in the ctor of derived classes)

Reply:
No problemo. That should be fine.